### ISSUE TITLE
Site overview rendered in large NGL view

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#618]
The 3D overview (top-left) is gone (thanks to the tags), but that view is still needed.

So this is a place-holder ticket for a way to get that rendered in the large NGL viewer.  Something as simple as a button "show site overview" somewhere along the top.

**Comment by mwinokan** [comment]
@phraenquex do you have any screenshots of what this view used to look like? I think it would be relevant for #1821

**Comment by phraenquex** [comment]
No, close this - come up with a new formalism, that old one was stale.
